nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/test/java/org/apache/nifi/controller/TestFlowController.java

```diff
@@ -117,12 +117,13 @@
     private BulletinRepository bulletinRepo;
     private VariableRegistry variableRegistry;
     private ExtensionDiscoveringManager extensionManager;
+    private StatusHistoryRepository statusHistoryRepository;
```

Comment:
Not necessary to make it in instance level attribute, as it is used only once

Reply:
Here it might be better because it is used twice: 183 and 434.

Reply:
I have no strong opinion, but this way adds a little bit of more code and complexity. In case of unit tests, it does not really matter if you create it twice, but makes it simpler. (Also I found it because other `FlowController` dependencies are created on the fly too, like `FlowRegistryClient`)